xrview/xrechnung.cpp

```diff
@@ -74,19 +74,19 @@ QDomDocument XRechnung::domDocument() const
 
 void XRechnung::createUbl()
 {
-    QStringList args;
-    QString file{ _url.toLocalFile() };
+    const QString file{ _url.toLocalFile() };
 
-    args.append("-jar");
-    args.append(SaxonJar);
-    args.append(QString("-s:%1").arg(file));
-    args.append(QString("-xsl:%1").arg(XslUBL));
-    args.append(QString("-o:%1").arg(ublFileName(_url)));
+    const QStringList args{
+        "-jar", 
+        SaxonJar,
+        QString("-s:%1").arg(file),
+        QString("-xsl:%1").arg(XslUBL),
+        QString("-o:%1").arg(ublFileName(_url)),
+    };
 
     QProcess *process = new QProcess;
 
-    connect(process, QOverload<int, QProcess::ExitStatus>::of(&QProcess::finished),
-                this, &XRechnung::slotDomFinished);
```

Comment:
This is to keep Qt5 compatibility, see https://stackoverflow.com/questions/12534367/qt-no-such-slot-for-the-qprocessfinished-signal that describes the problem. Are you sure your suggested change works with Qt5?

Reply:
Yes, when disabling deprecated public API the other signal is no longer visible and thus it is not overleaded.
(I have compiled this project with Qt5)

This is what allowed me to clean up lots of qOverloads in KDE :)